src/components/Experiment.vue

```diff
@@ -83,20 +117,47 @@ export default {
       }
       this.currentScreen += 1;
     },
+    /**
+     * Add a result set
+     * @public
+     * @param data{Object} a flat object whose data you want to add to the results
+     */
     addResult(data) {
       if (!this.results[this.currentScreen]) {
         this.results[this.currentScreen] = [];
       }
       this.results[this.currentScreen].push(data);
-    },
-    getRootElement() {
-      return this.$el;
     }
+  },
+  /**
+   * The contents of this slot will be visible during the entire experiment
+   * @slot title
+   */
+  /**
+   * Place your screens inside this slot. They will be visible one after the other, like a slide show.
+   * @slot screens
+   */
+  render(h) {
+    // HACKY-O
```

Comment:
I guess I'm still not totally sure of the use of a global `$exp` object. I think it 

- stores experiment results
- triggers `nextScreen()`

For the storage we can use a dedicated storage object, even if we don't really use Vuex. For the screen management I'm still thinking whether it would be possible to use some functionalities in Vue router. Though I understand that it might be tricky since this is not a classic use case.

Of course those could also be future refactoring opportunities to explore, and for now we can first put everything in one object and see how the app works out in general.

Reply:
Vue router would be interesting to use as well, although I think the current solution lends itself more closely to experiments. The idea was to have something similar to Vuex or Vue router that manages the experiment in one variable: $exp.